dynamic_programming/longest_increasing_subsequence.py

```diff
@@ -7,10 +7,21 @@
 Given an ARRAY, to find the longest and increasing sub ARRAY in that given ARRAY and return it.
 Example: [10, 22, 9, 33, 21, 50, 41, 60, 80] as input will return [10, 22, 33, 41, 60, 80] as output
 """
-
-
-def longestSub(ARRAY):  # This function is recursive
```

Comment:
I gotta say that I am not a fan of all these UPPERCASE VARIABLE NAMES as it runs counter to
https://pep8.org/#prescriptive-naming-conventions